framework_tvb/tvb/core/neotraits/_h5accessors.py

```diff
@@ -439,6 +439,28 @@ def load(self):
             return self.json_decoder().decode(val)
         return json.loads(val)
 
+class JsonRange(Scalar):
+    """
+    Stores and loads a Range in the form of a json in h5.
+    """
+
+    def __init__(self, trait_attribute, h5file, name=None, json_encoder=None, json_decoder=None):
+        super(JsonRange, self).__init__(trait_attribute, h5file, name)
+        self.json_encoder = json_encoder
+        self.json_decoder = json_decoder
```

Comment:
I don't think encoder and decoder are necessary here since we serialize a dictionary.